integrations/db/ojdbc/pom.xml

```diff
@@ -38,6 +38,12 @@
         <dependency>
             <groupId>com.oracle.ojdbc</groupId>
             <artifactId>ojdbc10</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.oracle.ojdbc</groupId>
```

Comment:
not sure this is right as ojdbc shouldn’t bring in ucp?

Reply:
it does as a transitivedependency

Reply:
```
[INFO] --- maven-dependency-plugin:3.1.2:tree (default-cli) @ ojdbc ---
[INFO] io.helidon.integrations.db:ojdbc:jar:2.0.0-SNAPSHOT
[INFO] \- com.oracle.ojdbc:ojdbc10:jar:19.3.0.0:compile
[INFO]    +- com.oracle.ojdbc:ucp:jar:19.3.0.0:compile
[INFO]    +- com.oracle.ojdbc:oraclepki:jar:19.3.0.0:compile
[INFO]    +- com.oracle.ojdbc:osdt_cert:jar:19.3.0.0:compile
[INFO]    +- com.oracle.ojdbc:osdt_core:jar:19.3.0.0:compile
[INFO]    +- com.oracle.ojdbc:simplefan:jar:19.3.0.0:compile
[INFO]    \- com.oracle.ojdbc:ons:jar:19.3.0.0:compile
```